test/core/storage/repository/in_memory_repository_test.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "storage/repository/impl/in_memory_repository.hpp"
+
+#include "storage/repository/repository_error.hpp"
+#include "testutil/literals.hpp"
+#include "testutil/outcome.hpp"
+#include "testutil/storage/base_fs_test.hpp"
+
+using fc::sector_storage::stores::LocalPath;
+using fc::sector_storage::stores::StorageConfig;
+using fc::storage::repository::InMemoryRepository;
+
+class InMemoryRepositoryTest : public test::BaseFS_Test {
```

Comment:
This test doesn't require file system, please remove inheritance.
As there is no members/methods in class that will be shared on test case, class can be removed (TEST should be used instead of TEST_F).